config/locales/en.yml

```diff
@@ -508,3 +509,14 @@ en:
         one: Displaying <b>1</b> %{model}
         other: Displaying <b>all&nbsp;%{count}</b> %{model}
         zero: No %{model} found
+  security_keys:
+    index:
+      title: Security keys
+      none: You don't have any security key
+      add: Add a security key
+    security_key:
+      delete: Delete
```

Comment:
Nitpick: it would be good to include the subject, so "Delete security key" instead of "Delete".

Reply:
done in 47e23ee